op-node/metrics/metrics.go

```diff
@@ -331,6 +334,12 @@ func NewMetrics(procName string) *Metrics {
 			"direction",
 		}),
 
+		ChannelInputBytes: factory.NewGauge(prometheus.GaugeOpts{
```

Comment:
With a gauge you'll only find the latest scraped number in the metrics. A data tx alone may contain more than one channel.
To get accurate metrics we should use a histogram, or a counter that only ever increases (and graph the rate of increase)

Reply:
You're right. Counter makes better sense here. Will fix.